### DISCUSSION TITLE
Transposing

### DISCUSSION THREAD

**Discussion by dynoweb** [jgadsden/tin-whistle-tablature#46] [Q&A]
When a staff is transposed, the tabs do not update. Is there a way to remove the tabs, transpose them, and then add them back in for the new instrument?

Or a clearer way to ask, how do I remove tabs?

**Comment by jgadsden** [comment]
Hello @dynoweb - I am not too sure, even though @DLLarson and I wrote most of the code
I will have a look and see what can be done
